cyto_dl/models/im2im/utils/postprocessing/auto_thresh.py

```diff
@@ -23,7 +23,10 @@ def thresh_func(image):
         self.thresh_func = thresh_func
 
     def __call__(self, image):
-        image = image.detach().cpu().float().numpy()
+        # Only get numpy array from torch tensor if tensor is passed in
+        # Allows np.ndarray to be passed in directly
+        if not isinstance(image, np.ndarray):
```

Comment:
Im not a fan of reassigning an arg (I know this was already happening before the change).  Could you use a new local var?